### ISSUE TITLE
[Bug]: Banner resolution for 3DS is wrong.

### ISSUE THREAD

**Issue by piplupness** [PretendoNetwork/juxtaposition#33]
### Checked Existing

- [X] I have checked the repository for duplicate issues.

### What happened?

When trying to add new communities, the community tab is reporting 400x220, but that's not the correct resolution.
Original Miiverse banners on 3DS used 400x168 images, but even if we decide to not use original Miiverse banners, it's still the wrong resolution overall, since Juxtaposition has still the community title at the bottom of the top screen, similar to Miiverse.

This bug can be circumvented by adding dummy pixels to 400x168 images to make them 400x220, and allowing setting banners on 3DS as intended. Setting 400x168 images currently just "zooms" the image.

### What did you expect to happen?

Image resolution standard being 400x168 for Nintendo 3DS.

### Steps to reproduce?

1. If your PNID is set to accessLevel = 3 (Developer), go to Moderation
2. Click Communities
3. Add a new community, as you can see, the resolution reported is 400x220! (Keep in mind, the 3DS screen is 400x240, so this should be already clear it's not the right one) 

### Other relevant information. (OPTIONAL)

This bug has been encountered when @MatthewL246 was testing and setting up new communities.

The current tab is this:
![image](https://github.com/user-attachments/assets/36a6e41b-c539-4dcf-adf0-9d306ef53c63)

**Comment by ashquarky** [comment]
![image](https://github.com/user-attachments/assets/a7dbc3fc-b793-4d25-9d77-e53f2560e2cc)

I believe the intention was for the title area to be semi-transparent, showing the remaining pixels through. However, if there are now communities with dummy pixels below the visible area, that change is no longer possible.

This is not related to the size of Miiverse's original images. Remember, Juxt is an all-new design and shares no code lineage to the original Miiverse design. 168 happens to be "close enough" to the 165px viewport, but this is coincidence.

**Comment by ashquarky** [comment]
> Also similar issue with Browser Icons... original Miiverse icons are 48x48

They're resized to 32, 64, and 128px versions on the server side after upload. Currently, Juxt displays the 64px version at 40x40px (in the title) and 60x60px (in the community list) on 3DS.
We need the higher resolution icons since they're also displayed on the Juxt web and Wii U frontends.

**Comment by piplupness** [comment]
> ![image](https://github.com/user-attachments/assets/a7dbc3fc-b793-4d25-9d77-e53f2560e2cc)
> 
> I believe the intention was for the title area to be semi-transparent, showing the remaining pixels through. However, if there are now communities with dummy pixels below the visible area, that change is no longer possible.
> 
> This is not related to the size of Miiverse's original images. Remember, Juxt is an all-new design and shares no code lineage to the original Miiverse design. 168 happens to be "close enough" to the 165px viewport, but this is coincidence.

Yep, we already have 400x220 banners from Archiverse and added the dummy pixels temporarily until it's 400x168.

**Comment by ashquarky** [comment]
My current plan for this is:
- Tweak the title area height so the banner viewport is now 168px
- Handle both 168px and 220px banners on the server
- 168px banners are extended down to 220px with white pixels (server-side)

I think this provides good compatibility with 168px assets while still allowing the transparency effect later if we want.
